src/main/webapp/css/home.css

```diff
@@ -15,3 +16,20 @@
 .error-message {
   padding: 5% 15%;
 }
+
+.search-bar {
+  background-color: #f7f7f7;
+  padding-bottom: 0.8%;
+  padding-top: 0.3%;
```

Comment:
From the style guide:

> Using shorthand properties is useful for code efficiency and understandability.

https://google.github.io/styleguide/htmlcssguide.html#Shorthand_Properties